src/reducers/index.js

```diff
@@ -0,0 +1,9 @@
+import { combineReducers } from 'redux';
+import books from './book';
+import { createBook, removeBook } from '../actions/index';
+
+export default combineReducers({
+  books,
+  createBook,
```

Comment:
`createBook` and `removeBook` are **Redux actions** not reducers. The only reducer you should create in the reducer folder and combine with books here is the category `filter` reducer.